constants/index.js

```diff
@@ -25,15 +25,15 @@ export const INCENTIVE_KEY = [
     // CHAIN_ID=1
     // INFURA='4c62211c73c248ebb0b6df4c999e52d9'
     // SWIV_TOKEN='0xbf30461210b37012783957D90dC26B95Ce3b6f2d'
-    // UNI_POOL=''
-    // START=1647105630
-    // END=1647205617
+    // UNI_POOL='0x387292c90500C16C9fE1F5A16Ce520677deC507f'
+    // START=1647622800
+    // END=1650301200
     // REFUND='0x173033758E8623fEE7C612e2f251CEa808127654'
-    // ETH_USDC_POOL='0x8ad599c3A0ff1De082011EFDDc58f1908eb6e6D8'
+    // USDT_USDC_POOL='0x3416cF6C708Da44DB2624D63ea0AAef7113527C6'
     // STAKER='0xe34139463bA50bD61336E0c446Bd8C0867c6fE65'
     // POSITIONS='0xc36442b4a4522e871399cd717abdd847ab11fe88'
     // NFTBATCHER='0xF83eEE39E723526605d784917b6e38ebCF0f0207'
-    // WETH='0xc778417E063141139Fce010982780140Aa0cD5Ab'
+    // USDC='0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48'
```

Comment:
Just to understand it right: on rinkeby it uses `WETH` but on mainnet `USDC`? 

I find it a bit dangerous that the actual exported constant is named `WETH` and comes from the env var `process.env.NEXT_PUBLIC_WETH`. This has a lot of potential to be misunderstood, when on mainnet this is actually `USCD`. It would probably be better to use a more neutral variable name here that isn't as confusing. 

If we are never gonna update/maintain this app, I would be sort of fine with it (maybe a comment and a remark in the README should at least be added) but ideally, you should consider renaming this var. This can be done quite easily in VS code by right clicking the variable name in `constants/index.js` and selecting "Rename" from the context menu - this will update the variable name in all referenced files as well (you don't have to search replace or anything...)

Reply:
Yeah I actually started doing that *right* after I made the PR lol, buut man that vscode shortcut is a lifesaver